crates/ide/src/hover/render.rs

```diff
@@ -500,3 +498,94 @@ fn local(db: &RootDatabase, it: hir::Local) -> Option<Markup> {
     };
     markup(None, desc, None)
 }
+
+struct KeywordHint {
+    description: String,
+    documentation: String,
```

Comment:
Nit: a better name would be something like `documented_module_name`/`keyword_mod` or some variation